config/locales/en/steps.yml

```diff
@@ -498,6 +498,10 @@ en:
             one: You have added %{count} asset
             other: You have added %{count} assets
       frozen_income_savings_assets_capital: *FROZEN_INCOME_ASSETS_PAGE_TEXT
+      answers:
+        edit:
+          page_title: Check your answers - Your client's capital - Apply for criminal legal aid - GOV.UK
```

Comment:
We don't usually have `- Apply for criminal legal aid - GOV.UK` on the end of our page titles 🤔 would be good to ask rachel why this has been added

Reply:
It gets added automatically @hiboabd, so yes @ripan " - Apply for criminal legal..." needs removing here.